app/modules/missions/models.py

```diff
@@ -538,17 +548,29 @@ def user_is_assigned(self, user):
     def get_assigned_users(self):
         return [assignment.user for assignment in self.user_assignments]
 
+    def get_assigned_users_with_assigner_json(self):
```

Comment:
always slightly twitchy with hand rolled json. Souldn't this be done by a schema?

Reply:
agreed. i couldnt think how to do it.  basically this _was_ (and still kind of is) the **schema for a list of users**.  however, for each user there is added a property called `assigner` which is _itself_ a user(!)... but this user comes from the **assignment** that designated the top-level user to be attached to this task.  i could not figure out how to get the schema to be passed this extra bit of information.   (may take up in slack with you - would love to know the magic to make this happen.)